### ISSUE TITLE
Warnings when running reproduce_results

### ISSUE THREAD

**Issue by bardhh** [Shaddadi/veritex#22]
There are two warnings that pop up. 
...
[2022-05-03 20:00:16,693][INFO] After 25 epochs / 0m 29s (29.440 seconds), with 0.9774 accuracy on test set.
[2022-05-03 20:00:16,693][INFO] Eventually the trained network got certified? True
[2022-05-03 20:00:16,694][INFO] === Avg <epochs, train_time, certified, accuracy> for 35 networks:
[2022-05-03 20:00:16,694][INFO] tensor([25.2857, 28.8715,  0.9714,  0.9473])
[2022-05-03 20:00:16,695][INFO] Total Cost Time: 1010.9472537349939s.

figure2_generator.py:54: UserWarning: Attempted to set non-positive bottom ylim on a log-scaled axis.
Invalid limit will be ignored.
  plt.ylim([0, 116])
/usr/local/lib/python3.7/site-packages/onnx2pytorch/convert/operations.py:154: UserWarning: The given NumPy array is not writeable, and PyTorch does not support non-writeable tensors. This means you can write to the underlying (supposedly non-writeable) NumPy array using the tensor. You may want to copy the array to protect its data or make it writeable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at  ../torch/csrc/utils/tensor_numpy.cpp:189.)
  weight = torch.from_numpy(numpy_helper.to_array(params[0]))

**Comment by Shaddadi** [comment]
The first issue is fixed. The second issue comes from a python package
